tools/ci_build/github/azure-pipelines/orttraining-win-gpu-ci-pipeline.yml

```diff
@@ -20,4 +20,4 @@ jobs:
     sln_platform: 'x64'
     CudaVersion: '10.1'
     OrtPackageId: 'Microsoft.ML.OnnxRuntime.Gpu'
-    BuildConfigurations: ['Debug', 'RelWithDebInfo']
+    BuildConfigurations: ['RelWithDebInfo']
```

Comment:
this change disables more than just a specific set of slow tests. can we be more fine-grained about it?

Reply:
Sure